conda/recipe/meta.yaml

```diff
@@ -64,17 +64,17 @@ requirements:
   host:
     - python
     - future >=0.17.1
-    - protobuf =3.15.8
-    - libprotobuf-static =3.15.8
-    - libjpeg-turbo >=2.0.5 # latest available is 2.0.5
+    - protobuf >=3.17.1
+    - libprotobuf-static >=3.17.1
```

Comment:
Using `>=` will work until it stops like last time :P (Just a comment, not a request to change).